nrf_wifi/fw_if/umac_if/inc/fmac_api_common.h

```diff
@@ -358,6 +358,22 @@ enum nrf_wifi_status nrf_wifi_fmac_set_packet_filter(void *dev_ctx, unsigned cha
 						     unsigned short buffer_size);
 #endif /* CONFIG_NRF700X_RAW_DATA_RX || CONFIG_NRF700X_PROMISC_DATA_RX */
 
+/**
+ * @brief Issue a request to get stats from the RPU.
+ * @param fmac_dev_ctx Pointer to the UMAC IF context for a RPU WLAN device.
+ * @param op_mode RPU operation mode.
+ * @param stats Pointer to memory where the stats are to be copied.
+ *
+ * This function is used to send a command to
+ *	    instruct the firmware to return the current RPU Lockup recovery statistics. The RPU will
+ *	    send the event with the current statistics.
```

Comment:
Unnecessary white space at the start of the comments

Reply:
It's part of format. Every comments have same white space.